src/otx/core/model/entity/base.py

```diff
@@ -397,3 +404,42 @@ def __iter__(self) -> Iterator:
         openvino.save_model(compressed_model, output_model_path)
 
         return output_model_path
+
+    def _generate_ptq_config(self, ov_model: Model) -> dict:
```

Comment:
This function rather reads PTQ config from IR, although, in a bit sophisticated way

Reply:
-> dict[str, Any]